ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+| paymaster                  | DATA, 20 Bytes  | Address of the Paymaster if it is paying for the transaction, `null` otherwise |
+| deployer                   | DATA, 20 Bytes  | Address of the Deployer if it is included in the transaction, `null` otherwise |
+| senderCreationGasUsed      | QUANTITY        | The amount of gas actually used by the sender deployment frame                 |
+| senderValidationGasUsed    | QUANTITY        | The amount of gas actually used by the sender validation frame                 |
+| paymasterValidationGasUsed | QUANTITY        | The amount of gas actually used by the paymaster validation frame              |
+| executionGasUsed           | QUANTITY        | The amount of gas actually used by the execution frame                         |
+| postOpStatus               | QUANTITY        | 1 (success), 0 (failure), or `null` (did not run) status of the `postOp` frame |
+| postOpGasUsed              | QUANTITY        | The amount of gas actually used by the paymaster `postOp` frame                |
+| validationLogs             | ARRAY           | Array of log objects, which this transaction'S VALIDATION FRAME generated.     |
+
+Continued, these fields are shared by all transaction types:
+
+| Name              | Type            | Value                                                                                      |
+|-------------------|-----------------|--------------------------------------------------------------------------------------------|
+| transactionHash   | DATA, 32 Bytes  | Hash of the transaction.                                                                   |
+| transactionIndex  | QUANTITY        | Integer of the transactions index position in the block.                                   |
+| blockHash         | DATA, 32 Bytes  | Hash of the block where this transaction was in.                                           |
+| blockNumber       | QUANTITY        | Block number where this transaction was in.                                                |
+| cumulativeGasUsed | QUANTITY        | The total amount of gas used when this transaction was executed in the block.              |
+| effectiveGasPrice | QUANTITY        | The sum of the base fee and tip paid per unit of gas.                                      |
+| gasUsed           | QUANTITY        | The amount of gas used by this specific transaction alone.                                 |
+| contractAddress   | DATA, 20 Bytes  | The contract address created, if the transaction was a contract creation, otherwise `null` |
+| logs              | ARRAY           | Array of log objects, which this transaction'S EXECUTION FRAME generated.                  |
+| logsBloom         | DATA, 256 Bytes | Bloom filter for light clients to quickly retrieve related logs.                           |
+| type              | QUANTITY        | Integer of the transaction type                                                            |
```

Comment:
not sure where to write it (probably, in eth_sendTranscation): we should specify that "type" field is mandatory when sending transaction. missing "type" implies eip-1559 (or legacy) transaction. Our code also should not depend on the presence of "sender", but on "type==AA_TX_TYPE"

Reply:
What is wrong with the `sender` check? I can see that `type` is not even an option in the `TransactionArgs` struct, meaning that no other transaction type relies on it in `eth_sendTranscation` for detection.